src/Components/Modal/GameInfo.js

```diff
@@ -61,11 +61,17 @@ render(){
                     style={{ marginBottom: '1em' }}
                 />
 
-                <button type="submit">Sign In</button>
-             </form>
+                <ul>
```

Comment:
Labeling on the homepage changed to allow running the front-end tests to run successfully.